code/modules/mob/living/silicon/mommi/death.dm

```diff
@@ -31,32 +29,3 @@
 
 	dead_mob_list -= src
 	qdel(src)
-
-/mob/living/silicon/robot/mommi/death(gibbed)
```

Comment:
this was literally a 1:1 copypaste of /mob/living/silicon/robot/death(), and it STILL called the parent anyways so it did everything twice